moduleframework/avocado_testers/avocado_test.py

```diff
@@ -233,7 +233,18 @@ def getIPaddr(self, *args, **kwargs):
 
         :return: str
         """
-        return self.backend.getIPaddr(*args, **kwargs)
+        warnings.warn("Function getIPaddr is deprecated. Use self.ipaddr instead",
+                      DeprecationWarning)
+        return self.backend.ipaddr(*args, **kwargs)
+
+    def ipaddr(self, *args, **kwargs):
```

Comment:
should this be wrapped with `@property`?

Reply:
also the name doesn't comply to pep 8; I would personally name it `ip_address`; the other thing is IPv4 and IPv6

Reply:
I would not complicated for now. Let's start with ip_address and later on we can enhance it for IPv6.